### DISCUSSION TITLE
Captum for edge features

### DISCUSSION THREAD

**Discussion by antoni4040** [pyg-team/pytorch_geometric#6244] [Q&A]
Hello. Is there a simple way to use Captum for edge features? I have set up Captum for node features and it wasn't very hard, but I'm not sure if the "edge" option is for edge weights or edge features...

**Comment by rusty1s** [comment]
@RBendias @wsad1

**Comment by RBendias** [comment]
Hi, unfortunately it is currently not possible to get explanations for edge features.
